operator/v1/types.go

```diff
@@ -59,6 +63,23 @@ type OperatorSpec struct {
 	ObservedConfig runtime.RawExtension `json:"observedConfig"`
 }
 
+type LogLevel string
+
+var (
+	// Normal is the default.  Normal, working log information, everything is fine, but helpful notices for auditing or common operations.
+	Normal LogLevel = "Normal"
+
+	// Debug is used when something went wrong.  Even common operations may be logged, and less helpful but more quantity of notices.
+	Debug LogLevel = "Debug"
+
+	// Trace is used when something went really badly and even more verbose logs are needed.  Logging every function call as part of a common operation, to tracing execution of a query.
+	Trace LogLevel = "Trace"
+
+	// TraceAtAllCost is used when something is broken at the level of API content/decoding.  It will dump complete body content.  If you turn this on in a production cluster
+	// prepare from serious performance issues and massive amounts of logs.
+	TraceAtAllCost LogLevel = "TraceAtAllCost"
```

Comment:
That name is not acceptable, although I can't come up with anything better, yet.

Reply:
Maybe just `All`

Reply:
TraceAll ?